proto/rill/runtime/v1/resources.proto

```diff
@@ -301,6 +302,27 @@ message BucketExtractPolicy {
   uint64 files_limit = 4;
 }
 
+message Theme {
+  ThemeSpec spec = 1;
+  ThemeState state = 2;
+}
+
+message ThemeSpec {
+  message Color {
+    float red = 1;
+    float green = 2;
+    float blue = 3;
+    float alpha = 4;
+  }
```

Comment:
Feels like a generic type, so maybe pull it out into a standalone generic type in `runtime/v1/color.proto`?